html_tag_collector/collector.py

```diff
@@ -166,12 +169,15 @@ async def get_response(session, url, index):
         # or the response is an unreadable content type
         # or the response code from the website is not in the 200s
         if (
-            response is not None and len(response.content) > 10000000
-            or content_type is not None and any(
+            response is not None
+            and len(response.content) > 10000000
+            or content_type is not None
+            and any(
                 filtered_type in content_type
                 for filtered_type in ["pdf", "excel", "msword", "image", "rtf", "zip", "octet", "csv", "json"]
             )
-            or response is not None and not response.ok
+            or response is not None
+            and not response.ok
```

Comment:
With a multi-condition if conditional like this, recommend extracting it to its own function, e.g. `response_valid()`